Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
```

Comment:
I'm 99% sure mounting a volume with the keys at runtime overrides what's set at build time, so this should be fine